superagent-mock-config.js

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable camelcase */
```

Comment:
This file is currently placed in the repo root, we'll want it in `test/test-helpers` instead.